app/stores/ada/HWVerifyAddressStore.js

```diff
@@ -96,30 +96,26 @@ export default class AddressesStore extends Store {
 
   ledgerVerifyAddress = async (
     path: BIP32Path,
+    address: string,
```

Comment:
Is the address being included for any particular reason? The verification doesn't seem to use the address at all, only the HD path. Do you know if it's possible to get the device address from Ledger after invoking `showAddress`? I ask because in the case the address shown on the device doesn't match the one in Yoroi there is no error shown anywhere.

Reply:
>Is the address being included for any particular reason? The verification doesn't seem to use the address at all, only the HD path.

When you press verify button in Yoroi, new page will be shown.
So on new page `address` is shown there so user could verify manually (if we do not show `address` on new page then user has to come back to Yoroi and check the `address` which is not a good UX)

>Do you know if it's possible to get the device address from Ledger after invoking showAddress? I ask because in the case the address shown on the device doesn't match the one in Yoroi there is no error shown anywhere.

https://github.com/cardano-foundation/ledgerjs-hw-app-cardano/blob/ac3ee1345506ab343a7159ebbcec8e616f8ac5d9/src/Ada.js#L388
Not possible ATM.